python/main.py

```diff
@@ -62,15 +60,21 @@ def root():
 
 
 @app.post("/items")
-def add_item(name: str = Form(...), category: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...), image: str = Form(...)):
```

Comment:
Could you see [this PR](https://github.com/mercari-build/mercari-build-training-2022/pull/63/files) and update your code if you have time.
Instead of sending the file name, you could send the image file itself 🙂

Reply:
Yes, since I've submitted this PR before fetching upstream, it was not adequately updated. Just updated now!